src/lib/plugins/entities.js

```diff
@@ -76,6 +73,16 @@ module.exports.server=function(serv,options) {
         if (entity.deathTime && Date.now() - entity.bornTime >= entity.deathTime) {
           entity.destroy();
           return;
+        } else if (entity.pickupTime && Date.now() - entity.bornTime >= entity.pickupTime) {
+          var players = serv.getNearby({
+            world: entity.world,
+            position: entity.position,
+            radius: 1.5*32 // Seems good for now
+          });
+          if (players.length) {
```

Comment:
Why is that check needed ? UpdateAndSpawn already handle counting players and all.